### DISCUSSION TITLE
More customization options required for typography

### DISCUSSION THREAD

**Discussion by fraktalize** [heroui-inc/heroui#708] [Feedback]
Hey! 
Loving NextUI, really beautifully made and easy to use. I like that it's based on Stitches as well.

When customizing the theme I'm facing some issues that I can easily circumvent using the `css` prop or global styles, but which I feel might be benificial to have working inside NextUI.

The first one is: I need to be able to use a different typeface for the headings. Currently I can only set it through CSS props - I would like to be able to set the fonts using the variables in the Stitches theme, maybe something like: `<Text h1 font="$display">`

Also, setting the line height directly in the `Text` component would be great - the default isn't doing it for the typefaces I'm using.

Thanks for a great UI lib!

Regards,
Typography Nerd

**Comment by fraktalize** [comment]
Oh and another thing that would be really great is the ability to set different font sizes for different media queries. Right now I need to manually set it with css and it's a bit messy :)

**Comment by jrgarciadev** [comment]
Hey please update to v2.0 https://nextui.org/blog/nextui-v2, we introduced multiple customization options and with TailwindCSS is a lot easier.

**Comment by leetcow** [comment]
hey guys, how can I customize the font family on latest heroUI? can't seem to find the option in the docs
